### DISCUSSION TITLE
Any updates on Android 14?

### DISCUSSION THREAD

**Discussion by EfeWasHere** [AndreyPavlenko/Fermata#426] [General]
Hello, I have a Samsung Galaxy A54 5G with Android 14/OneUI 6.0. I have installed Fermata Auto 1.9.9 (with KingInstaller) on my phone and turned on the "unknown sources" option in AA settings but only "Fermata Media Service" is shown on the app list on AA (which does basically nothing). I know this is a known thing, but I just wanted to ask if there are any updates on this situation (or if there are any workarounds).

Thanks in advance.

**Comment by AndreyPavlenko** [comment]
The possible workarounds - https://github.com/AndreyPavlenko/Fermata/issues/327#issuecomment-1934257166

**Comment by TalSivan** [comment]
> The possible workarounds - https://github.com/AndreyPavlenko/Fermata/issues/327#issuecomment-1934257166

Please tell me what we need to do ?
To my friend have s24 + android 14, after update to 1.9.9 its working.

**Comment by AndreyPavlenko** [comment]
The easiest way is to buy AAWireless.

**Comment by TalSivan** [comment]
> The easiest way is to buy AAWireless.

I do not understand you, in my car i dont need Nothing, The connection to AA is wireless.
Why S24 working good with android 14 afer update to 1.9.9

**Comment by dreimer1986** [comment]
The easiest way is to buy AAWireless.

Why? Because this way you can force the signal into development mode and thus the car screen shows Fermata as it was going to in A13. If that is not what you want, then the Workarounds https://github.com/AndreyPavlenko/Fermata/issues/327#issuecomment-1934257166 are what you need. Read them and do some own research. No one here plans to spoon feed you to a solution just that the next lil hiccup will make you stand here again because you never learned to tinker yourself.

**Comment by TalSivan** [comment]
> The easiest way is to buy AAWireless.
> 
> Why? Because this way you can force the signal into development mode and thus the car screen shows Fermata as it was going to in A13. If that is not what you want, then the Workarounds https://github.com/AndreyPavlenko/Fermata/issues/327#issuecomment-1934257166 are what you need. Read them and do some own research. No one here plans to spoon feed you to a solution just that the next lil hiccup will make you stand here again because you never learned to tinker yourself.


Thanx for your Answer, but im very Stupid 🤷🏻‍♂️ in #327 have many comment im lost ther.. please tell me what i need to do ?
Dongel not good for me 🥺

**Comment by dreimer1986** [comment]
I am not close to my PC right now, but first thing. I never said you are stupid ^^ I agree that the DHU trick is complicated, but according to others it works. I can not confirm that as I have a AAWireless for my car. But back to the main topic. There are two ways to get a less restricted Android Auto Display where you can select Fermata Widescreen as "Navigation" app. This will allow you to get back to your car and then there the last used navigation app is started automatically. Problem is if you ever change to Google maps, you have to redo this trick. If you want to have things a bit easier and without installing the whole Google Development kit, there is Headunit Reloaded Emulator, too. It's on play store, but costs money. This installed should show as "car" in Android Auto and you can bypass the app restriction this way.

**Reply by maripic69** [reply]
Здравей колега, искам да попитам ако го купя това Headunit Reloaded Emulator, какво да направя да тръгне фермата ауто на дисплея на колата с телефон самсунг галакси е с 24 ултра ONE UI 6.1? Само да спомена че съм с безжичен андроид ауто!  Благодаря!!!

**Comment by TalSivan** [comment]
> I am not close to my PC right now, but first thing. I never said you are stupid ^^ I agree that the DHU trick is complicated, but according to others it works. I can not confirm that as I have a AAWireless for my car. But back to the main topic. There are two ways to get a less restricted Android Auto Display where you can select Fermata Widescreen as "Navigation" app. This will allow you to get back to your car and then there the last used navigation app is started automatically. Problem is if you ever change to Google maps, you have to redo this trick. If you want to have things a bit easier and without installing the whole Google Development kit, there is Headunit Reloaded Emulator, too. It's on play store, but costs money. This installed should show as "car" in Android Auto and you can bypass the app restriction this way.


Thank you very much ! I appreciate it !
last question if its ok...
I have another phone with android 13, its possible to see tv in fermata auto ? 
What i need to do ? 
(Sorry about my english)

**Comment by AndreyPavlenko** [comment]
For Android 13 just use this installer - https://github.com/fcaronte/KingInstaller/releases

**Comment by divebombdom** [comment]
I keep seeing AAWireless referenced. Is this an absolute or does this method work with the Motorola equivalent?

**Reply by fcaronte** [reply]
No, won't work because AAwireless have a special setting to unlock this all "developer mode"

**Comment by adventuryone** [comment]
Buongiorno a tutti coloro interessati al funzionamento di Fermata auto su android 14. Ho acquistato AAWireless e con S21 sono andato nelle impostazioni del dongle e ho attivato modalità sviluppatore e disattivato come chiede lo stesso AAWireless l'impostazione passthrough. Funziona benissimo spero vi sia d'aiuto

**Comment by dreamumut** [comment]
Easy solution https://t.me/Androidautofermata
